contracts/contract_state_tree.md

```diff
@@ -13,20 +14,23 @@ The contract state tree contains objects:
 - The contract log (data binary)
 - Active flag (Boolean)
 - List of refering contracts
-- Calls & Return data
 - Deposit (amount)
 - vm_version
 
-As well as normal account fields.
+See [Contract Serialization](../serializations.md#contract) for the serialization specification.
```

Comment:
It looks like a nonce is missing in the doc https://github.com/aeternity/epoch/blob/098fcfad68adc13fac7842103de299c20ffbc22d/apps/aecontract/src/aect_contracts.erl#L187